BuildScript/BuildTasks/StringEnumerableExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Remotion.BuildScript.BuildTasks
+{
+  public static class StringEnumerableExtensions
+  {
+    public static IEnumerable<string> DuplicateValues (this IEnumerable<string> enumerable, StringComparer comparer = null)
```

Comment:
GetDuplicateValues, otherwise, it would be "duplicate the values in the list"

Reply:
I was actually thinking about this, stuff like `First()`, `Distinct()`, `Max()` etc. als don't have a `Get` prefix, which is why I went with that name. Are you referring to re-motion's extension methods? I did find stuff like `ToX()` for conversions and `ApplyX` etc. but I couldn't find an instance of `GetX`

Reply:
Ah, true. Didn't think about First, Distinct and Max. Those are clear from their meaning (and probably to everyone familiar with SQL), they can't be used as a verb. The problem is, duplicate can be used as a verb and as an adjective. Maybe there's a better name?

Reply:
I see your point, but I can't think if a better name and the best thing the thesaurus gave me was `Indistinct()` as an antonym to `Distinct()`. I'd just go with `GetDuplicateValues` and carry on from there